lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -237,6 +237,39 @@ def _create_custom_template_with_hours_of_effort(self, org_id=None, mode=None, c
         )
         template.save()
 
+    def _create_custom_template_with_verified_description(self, org_id=None, course_key=None, language=None):
+        """
+        Creates a custom certificate template entry in DB. This custom certificate can be used to test
+        that the correct language is used if the integrity signature feature has been enabled for a course.
+        """
+        template_html = """
+            <%namespace name='static' file='static_content.html'/>
+            <html>
+            <body>
+                lang: ${LANGUAGE_CODE}
+                course name: ${accomplishment_copy_course_name}
+                mode: verified
+                ${accomplishment_copy_course_description}
+                ${certificate_type_description}
+                % if is_integrity_signature_enabled_for_course:
+                <p> Integrity signature enabled>
```

Comment:
Is there supposed to be a closing </p> after `enabled`? The ending `>` seemed weird at first glance.

Reply:
That is definitely a typo, thanks for catching that.